res/css/views/messages/_MVideoBody.scss

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2020 New Vector Ltd.
```

Comment:
Is that right for stuff in `matrix-react-sdk`?

Reply:
Hmm, you're right, pretty sure I copied it from MImageBody :')